examples/app/random.rs

```diff
@@ -0,0 +1,96 @@
+use bevy::prelude::*;
+use rand::{prelude::IteratorRandom, rngs::SmallRng, Rng, SeedableRng};
+
+/// This example illustrates how to use entropy to control randomness in bevy.
+/// We randomly choose a coin to toss and record the result.
+///
+/// Because all random number generators are seeded from the same "world seed",
+/// this program is deterministic.
+struct Coin;
+
+struct Toss(SmallRng);
```

Comment:
When do we want to use `SmallRng` vs the standard `Rng`?

Reply:
https://docs.rs/rand/0.8.4/rand/rngs/index.html
> SmallRng is an **insecure** PRNG designed to be fast, simple, require little memory, and have good output quality.

Reply:
Useful! The question was mostly rhetorical, intended as "we should have a small comment explaining this here".